frontend/components/puzzle4.jsx

```diff
@@ -34,9 +35,23 @@ class Puzzle4 extends React.Component {
     });
   }
 
+  submitAnswer() {
+    let answer = this.state.value;
```

Comment:
these don't look like these are reassigned, use `const` instead? i'll add to eslint

Reply:
yeah changing to const makes sense to me